spec/models/task_spec.rb

```diff
@@ -29,6 +30,13 @@
       expect(task).not_to be_valid
     end
 
+    it "raises validation error with try to save task without project" do
+      expect {
+        task = FactoryGirl.build(:task)
```

Comment:
You could move `task` initialization above `expect` to reduce its scope. Also, you can check for specific error to make sure it's exactly what you expect. Take a look: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher